liquid/intranet/quote/models.py

```diff
@@ -56,11 +58,14 @@ def save(self, *args, **kwargs):
          self.quote_source_html += "<a href='/intranet/quote/?author=" + author_netid + "'>" + author + "</a>" 
            
       # Hyperlink posters (people who edited the quote but didn't say anything noteworthy)
-      self.quote_posters = "," + html.escape(re.sub(",\s+", ",", self.quote_posters).strip(",")) + ","
-      posters = self.quote_posters.strip(",").split(",")
+      if len(self.quote_posters) != 0: # Posters were not recorded for some earlier quotes
+         self.quote_posters = "," + html.escape(re.sub(",\s+", ",", self.quote_posters).strip(",")) + ","
+         posters = self.quote_posters.strip(",").split(",")
+      else:
+         posters = dict() # Posters should contain no elements if self.quote_posters is blank
          
       # If the quote posters are different than the quote authors, create the quote poster HTML
-      posters_are_authors = len(posters) == len(authors);
```

Comment:
[http://cdn.meme.am/instances/500x/64783469.jpg](url)

...Whoops, let me fix that.